src/Common/src/TypeSystem/Common/TypeDesc.cs

```diff
@@ -485,7 +485,7 @@ public virtual IEnumerable<MethodDesc> GetMethods()
         /// is returned. Returns null if method not found.
         /// </summary>
         // TODO: Substitutions, generics, modopts, ...
-        public virtual MethodDesc GetMethod(string name, MethodSignature signature)
+        public virtual MethodDesc GetMethod(string name, MethodSignature signature = null)
```

Comment:
I would prefer we don't do this. This method won't throw anything if the match is ambiguous (we have perf reasons - as opposed to e.g. `System.Type::GetMethod(string)`) - I find it better to force people to make an explicit choice at the callsite.